css/properties/mask-size.json

```
@@ -6,10 +6,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/mask-size",
           "support": {
             "chrome": {
-              "version_added": false
+              "prefix": "-webkit-",
+              "version_added": "4"
             },
             "chrome_android": {
-              "version_added": false
+              "prefix": "-webkit-",
+              "version_added": "28"
```

I believe that this may be a typo and you meant “18”, since that’s the first Chrome Android version?

```suggestion
              "version_added": "18"
```

---

28 is what's in the Status entry. I don't know whether to tell you that's mistake or not. 18 is probably right, but I have no way to research it.

---

I'm leaning towards "18" being accurate as the feature was introduced in Safari 4, meaning it was introduced to WebKit, and I highly doubt that it's OS-dependent.  Though since that's what's in Chrome Status, we could perhaps leave it and come back to it another time, merging what we've got now?

---

I'm okay either way.

@jpmedley what's your preference? I don't think it will make much of a practical difference for consumers of the data since both of these are relatively "old" versions.

---

28 was when Blink forked from WebKit. It was released on June 25, 2013. If Safari 4 was released before that date use 18. If after, use 28.

---

Safari 4 was released in 2009, so I guess 18 it is!

---

(If we were working on it before the fork, it might still be prefixed after. In fact, we still have a few prefixed items.)

---

I take everything back. I forgot that **there was no Chrome 18**. As I've stated elsewhere, this is likely related to an issue with getting the Chrome feature to work on Android.

---

* No Chrome 18 on Android, that is.